src/ui/components/Difference/index.jsx

```diff
@@ -187,11 +265,52 @@ function Difference() {
   );
 }
 
+const Pagination = styled.div`
+  display: flex;
+  justify-content: space-between;
+
+  .pagination-content {
+    display: flex;
+    align-items: center;
+    margin-right: 10px;
+    width: 100px;
+
+    text-align: left;
+  }
+
+  .pagination-prev-button {
+    min-width: 140px !important;
+    margin-right: 10px;
+    border: 1px solid #adb5bd;
+
+    text-align: center;
+  }
+
+  .pagination-next-button {
+    border: 1px solid #adb5bd;
+    min-width: 140px !important;
```

Comment:
P1: width, height가 더 위로 올려주시면 좋을 것 같습니다 :) 
```suggestion
min-width: 140px !important;
border: 1px solid #adb5bd;
```